tests/flexalloc_tests_common.c

```diff
@@ -519,25 +520,24 @@ fla_ut_lpbk_fs_destroy(struct fla_ut_lpbk *lpbk, struct flexalloc *fs)
   return err;
 }
 
-int
-fla_ut_fs_use_device()
+bool
+is_globalenv_set(char const * glb)
 {
-  char *dev_uri = getenv("FLA_TEST_DEV");
-  return dev_uri != NULL;
+  return getenv(glb) != NULL;
 }
 
 int
 fla_ut_dev_init(uint64_t disk_min_blocks, struct fla_ut_dev *dev)
 {
   struct xnvme_dev *xdev;
   int err = 0;
-
-  dev->_dev_uri = getenv("FLA_TEST_DEV");
-  dev->_md_dev_uri = getenv("FLA_MD_DEV");
   dev->_is_zns = 0;
+  dev->_md_dev_uri = NULL;
 
-  if (dev->_dev_uri != NULL)
+  if(is_globalenv_set("FLA_TEST_DEV"))
   {
+    dev->_dev_uri = getenv("FLA_TEST_DEV");
+    dev->_md_dev_uri = getenv("FLA_TEST_MD_DEV");
```

Comment:
Should we use the is_globaenv_set helper here as a guard?

Reply:
The way things are, we do not need at as we check for null further down and getenv is sure to give us a NULL when it does not exists.